crates/rocksdb/src/db_manager.rs

```diff
@@ -301,6 +301,9 @@ impl RocksDbManager {
         //
         db_options.set_table_cache_num_shard_bits(6);
 
+        // Speed up database open, useful for large databases and slow disk.
+        db_options.set_skip_stats_update_on_db_open(true);
```

Comment:
I guess the statistics will then be updated as compactions are triggered and data is inserted into RocksDB, right?

Reply:
Yes. It's a _very_ common practice to disable it actually.